graylog2-web-interface/src/components/alarmcallbacks/AlarmCallbackHistoryOverview.jsx

```diff
@@ -30,7 +30,9 @@ const AlarmCallbackHistoryOverview = React.createClass({
       return <Spinner />;
     }
 
-    const histories = this.state.histories.map(this._formatHistory);
+    const histories = this.state.histories
+      .sort((h1, h2) => h1.created_at.localeCompare(h2.created_at))
```

Comment:
Comparing two timestamps lexicographically might not lead to a correct result. We should use `moment().isBefore()/isAfter()` for that.